docs/source/how_to_guides/configure_cloud_storage_cred.md

```diff
@@ -2,8 +2,11 @@
 
 Streaming dataset supports the following cloud storage providers to stream your data directly to your instance.
 - Amazon S3
+- S3 compatible object store
```

Comment:
Not alphabetical?

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/265/commits/de2d844124edd2be117f70455ca0edb8cddc5c23.